packages/experimental/NativeFontMetrics/ios/FRNFontMetrics.h

```diff
@@ -0,0 +1,31 @@
+#import <React/RCTBridgeModule.h>
+#import <React/RCTEventEmitter.h>
+
+@interface FRNFontMetrics : RCTEventEmitter <RCTBridgeModule>
+@end
+
+#import <React/RCTConvert.h>
+
+typedef NS_ENUM(NSInteger, FRNTextStyle) {
```

Comment:
If useful, there is a `RCT_ENUM_CONVERTER` that sets a default (I see you. do that in the .m file).

I use it in `FRNAVatarCiewManager.m`

Reply:
These enum values have to be defined somewhere first. The values in FRNAvatarViewManager.m are already defined in fluentui-apple.